chat_exporter/construct/message.py

```diff
@@ -27,7 +29,12 @@
 
 
 def _gather_user_bot(author: discord.Member):
-    return bot_tag if author.bot else ""
+    if author.bot and author.public_flags and author.public_flags.verified_bot:
```

Comment:
Is `author.public_flags` necessary? This doesn't return `None` afaik

Reply:
I honestly don't know it for sure. So I added it just in case

Reply:
I'd say return it then, API documentation doesn't say it returns `None`